app/src/main/java/co/techmagic/hr/presentation/util/ViewExtentions.kt

```diff
@@ -25,9 +25,10 @@ fun View.changeShapeStrokeColor(@DimenRes strokeWidth: Int, @ColorRes colorRes:
     } else {
         throw IllegalArgumentException()
     }
-    drawable.setStroke(
+
+    val newDrawable = drawable.mutate() as GradientDrawable
+    newDrawable.setStroke(
```

Comment:
Seems you don't need `newDrawable` variable but `mutate()` should be called before `setStroke(...)`